pom.xml

```diff
@@ -284,6 +289,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.maven.shared</groupId>
+      <artifactId>maven-invoker</artifactId>
+      <version>2.2</version>
```

Comment:
So you have one version 3.1.1 of maven-invoker and one version 2.2? Does it make sense?

Reply:
Probably not. This is a leftover from previous trials. I'll fix this ASAP.